index.html

```diff
@@ -20,14 +20,20 @@
 <body>
     <header class="nav-container">
         <div class="navbar-list">
-            <a href="#">
-                <h3 class="name1">AB</h3>
-            </a>
+            <h3 class="name1"><a href="#">AB</a></h3>
             <i class="hamburger-menu fa fa-bars"></i>
+            <ul class="hamburger-list">
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li>
+                    <a href="#"><i class="fa fa-envelope-o"></i> </a>
+                </li>
```

Comment:
- [ ] please make the headline section responsive. When switching to the desktop version, the text goes one-sided. Kindly fix and centralize it. 
- [ ] also make your heading bolder, so they look more like the figma version
your version 
![Screenshot (217)](https://user-images.githubusercontent.com/65298048/177863054-b0432152-ec2a-429d-838a-b449b0e81bac.png)
figma
![Screenshot (215)](https://user-images.githubusercontent.com/65298048/177863233-0557cc12-85c6-452d-972f-7d92d3016ed8.png)